repos/aspnetcore-tooling.proj

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <SourceDirectory>AspNetCore-Tooling</SourceDirectory>
+    <SourceDirectory>aspnetcore-tooling</SourceDirectory>
```

Comment:
Don't we generate this value from the versions file?

https://github.com/dotnet/source-build/blob/5c10e17ef8c990587077b10cb3221fa480e24b47/eng/Version.Details.xml#L36-L40